spellcheck.json

```diff
@@ -11,6 +11,5 @@
   "jewlry": "jewelry",
   "clik": "click",
   "clikc": "click",
```

Comment:
Reorder? Otherwise, the user's code would get broken if they had line 13's entry somewhere.

Reply:
Yea, that makes sense.